.github/workflows/test-pr.yml

```diff
@@ -13,29 +13,28 @@ jobs:
     name: 'Version Bump'
     runs-on: ubuntu-latest
     steps:
-      - name: 'Check out code, setup git/poetry'
+      - name: 'Check out code'
         uses: actions/checkout@v3
         with:
           token: ${{ secrets.JENKINS_GITHUB_PAT }}
           # fetch-depth 0 means deep clone the repo
           fetch-depth: 0
           ref: ${{ github.event.pull_request.head.sha }}
-      - run: |
+      - name: 'Configure GitHub user'
+        run: |
           git config user.name devops
           git config user.email devops@runtimeverification.com
-          curl -sSL https://install.python-poetry.org | python3 - --version 1.3.2
-          poetry --version
-      - name: 'Update Version'
+      - name: 'Install Poetry'
+        uses: Gr1N/setup-poetry@v8
+      - name: 'Update version'
         run: |
           og_version=$(git show origin/${GITHUB_BASE_REF}:package/version)
           ./package/version.sh bump ${og_version}
           ./package/version.sh sub
           git add --update && git commit --message "Set Version: $(cat package/version)" || true
       - name: 'Sync Poetry'
         run: |
-          cd kevm-pyk
-          poetry update
-          cd ..
+          poetry -C kevm-pyk update
```

Comment:
This step is probably unnecessary: the previous step updates the project version, but that has no effect on the lock file.

Reply:
For example `pyk` does not do this: https://github.com/runtimeverification/pyk/blob/60e476839632474fd11b578aed3f048804bc1d4a/.github/workflows/test-pr.yml#L21

Reply:
Removed.

@ehildenb, is the subsequent `Update nix flake inputs` necessary, i.e. does anything need to change in Nix files due to bumping `kevm-pyk` version?

Reply:
Well, it appears that updating the lock files is probably not ncesesary, though they definitely do seem to change on runs of this PR. But that's because the version-bumping section of tehh PR updates them anyway: https://github.com/runtimeverification/evm-semantics/blob/a68c97a2280b7ac9f1fc81f142464d0a23793b8d/.github/workflows/test-pr.yml#L12

So maybe they can be removed from this branch-level workflow, and we just sync the normal files here?

Reply:
Removed `Update nix flake inputs` as well.